src/main/daml/Daml/Finance/Interface/Settlement/Instruction.daml

```diff
@@ -62,7 +62,9 @@ interface Instruction where
     with
       allocation : Allocation
         -- ^ Allocation of an instruction.
-    controller (view this).step.sender
+      actors : Parties
```

Comment:
I think we often follow an unwritten convention to put the `actors` parameter as first one in a choice.

Reply:
agree, that looks better. I added it to the style guide.